lib/bypass/filter.rb

```diff
@@ -1,13 +1,25 @@
+# frozen_string_literal: true
+
 module Bypass
   class Filter
     attr_reader :content, :fragment
 
-    URL_PATTERN = /\bhttps?:\/\/
-      [a-zA-Z0-9\-\._~:\/\?#\[\]@!$&'\(\)\*\+,;=%]+
-      [a-zA-Z0-9\-_~:\/\?#\[\]@!$&\*\+;=%]/x
+    URL_PATTERN = %r{
+      # Start of a word
+      \b
+
+      # Protocol
```

Comment:
scheme

Reply:
Apparently the `//` starts the Authority section. TIL

I'll just dumb it down some.